### PR TITLE
Add test command in build time

### PR DESCRIPTION
**Pull request by MilesChou**
Run dev tools cil for test in build time.

See https://github.com/phalcon/phalcon-devtools/blob/master/phalcon.php

Also, I add a new maintainer. Thanks fizzka.

### CODE REVIEW

**Review by fizzka — APPROVED** [review]
looks good

File: 7.2/alpine/Dockerfile [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
-LABEL maintainer MilesChou <jangconan@gmail.com>
 
-ENV PHALCON_VERSION=3.3.1
-ENV PHALCON_DEV_TOOLS_VERSION=3.2.12
+LABEL maintainer="MilesChou <github.com/MilesChou>"
+LABEL maintainer="fizzka <github.com/fizzka>"
+
+ARG PHALCON_VERSION=3.3.1
+ARG PHALCON_DEV_TOOLS_VERSION=3.2.12
+ARG PHALCON_EXT_PATH=php7/safe
```
Comment (fizzka) [inline 7.2/alpine/Dockerfile]: 1) are you sure about "safe"?
2) i think i needs to do additional calculation for php5 (in common)

Comment (MilesChou) [reply 7.2/alpine/Dockerfile]: I not sure. I don't know the difference between "safe" and the others.

Should I use 64bits?

Comment (fizzka) [reply 7.2/alpine/Dockerfile]: it needs investigations...
i used 64bits, and phalcon native install script also uses 64bits (can be seen in build trace)

Comment (MilesChou) [reply 7.2/alpine/Dockerfile]: OK, I will follow Phalcon native install. :)

**Comment by MilesChou** [comment]
I adjust 7.x-alpine image, but 5.x-alpine is not work using `docker-php-ext-install`.

**Comment by fizzka** [comment]
could you show not working php5 dockerfile?
i'll try to find solution

2018-01-23 9:03 GMT+03:00 MilesChou <notifications@github.com>:

**Comment by MilesChou** [comment]
FYI:

```dockerfile
FROM php:5.5-alpine

LABEL maintainer="MilesChou <github.com/MilesChou>"
LABEL maintainer="fizzka <github.com/fizzka>"

ARG PHALCON_VERSION=3.3.1
ARG PHALCON_DEV_TOOLS_VERSION=3.2.12
ARG PHALCON_EXT_PATH=php5/64bits

RUN set -xe && \
        # Compile Phalcon
        curl -LO https://github.com/phalcon/cphalcon/archive/v${PHALCON_VERSION}.tar.gz && \
        tar xzf v${PHALCON_VERSION}.tar.gz && \
        docker-php-ext-install -j $(getconf _NPROCESSORS_ONLN) /cphalcon-${PHALCON_VERSION}/build/${PHALCON_EXT_PATH} && \

        # Insall Phalcon Devtools for testing, see https://github.com/phalcon/phalcon-devtools/
        curl -LO https://github.com/phalcon/phalcon-devtools/archive/v${PHALCON_DEV_TOOLS_VERSION}.tar.gz && \
        tar xzf v${PHALCON_DEV_TOOLS_VERSION}.tar.gz && \

        # Test Phalcon using Phalcon Devtools
        php /phalcon-devtools-3.2.12/phalcon.php && \

        # Remove all temp files
        rm -rf \
            v${PHALCON_VERSION}.tar.gz \
            /cphalcon-${PHALCON_VERSION} \
            v${PHALCON_DEV_TOOLS_VERSION}.tar.gz \
            phalcon-devtools-${PHALCON_DEV_TOOLS_VERSION}
```

It will be failure at `docker-php-ext-install` command. 

BTW, `5.6-alpine` is okay.

**Comment by fizzka** [comment]
# Insall Phalcon Devtools -> # Install Phalcon Devtools
typo everywhere (copypaste)

i'll check docker-php-ext-install


2018-01-23 9:48 GMT+03:00 MilesChou <notifications@github.com>:

**Comment by fizzka** [comment]
reason why php5.5 fails build is docker-php-ext-install prior 5.6 cant use external path as extension.
my workaround here (docker-php-source extract & symlink):

```
FROM php:5.5-alpine

LABEL maintainer="MilesChou <github.com/MilesChou>"
LABEL maintainer="fizzka <github.com/fizzka>"

ARG PHALCON_VERSION=3.3.1
ARG PHALCON_DEV_TOOLS_VERSION=3.2.12
ARG PHALCON_EXT_PATH=php5/64bits

RUN set -xe && \
        # Compile Phalcon
        curl -LO https://github.com/phalcon/cphalcon/archive/v${PHALCON_VERSION}.tar.gz && \
        tar xzf v${PHALCON_VERSION}.tar.gz && \
        docker-php-source extract && \
        ln -s /cphalcon-${PHALCON_VERSION}/build/${PHALCON_EXT_PATH} /usr/src/php/ext/phalcon && \
        docker-php-ext-install -j $(getconf _NPROCESSORS_ONLN) phalcon && \
        docker-php-source delete &&\

        # Insall Phalcon Devtools for testing, see https://github.com/phalcon/phalcon-devtools/
        curl -LO https://github.com/phalcon/phalcon-devtools/archive/v${PHALCON_DEV_TOOLS_VERSION}.tar.gz && \
        tar xzf v${PHALCON_DEV_TOOLS_VERSION}.tar.gz && \

        # Test Phalcon using Phalcon Devtools
        php /phalcon-devtools-3.2.12/phalcon.php && \

        # Remove all temp files
        rm -rf \
            v${PHALCON_VERSION}.tar.gz \
            /cphalcon-${PHALCON_VERSION} \
            v${PHALCON_DEV_TOOLS_VERSION}.tar.gz \
            phalcon-devtools-${PHALCON_DEV_TOOLS_VERSION}
```

**Comment by MilesChou** [comment]
Great!! I use your revision to commit.

I will merge Alpine image first if CI server has no problems.

**Comment by fizzka** [comment]
need to same workarounds for debian-based images?

**Pull request merged**